src/Sarif.Viewer.VisualStudio/ErrorList/ErrorListService.cs

```diff
@@ -514,6 +527,27 @@ private int WriteRunToErrorList(Run run, string logFilePath)
             return sarifErrors.Count;
         }
 
+        private void ShowFilteredSuppressionStateColumn()
+        {
+            // Creating this table source adds "Suppression State" to the list of available columns.
+            SarifSuppressedResultsTableDataSource _ = SarifSuppressedResultsTableDataSource.Instance;
```

Comment:
These should no-op if the column has already been added (if they are not already doing that).

Reply:
Yes, that is the behavior I expected and observed. Were you suggesting a code change, or just making sure I understood the implications of this line?

---
In reply to: [537999155](https://github.com/microsoft/sarif-visualstudio-extension/pull/313#discussion_r537999155) [](ancestors = 537999155)